### PR TITLE
Create  ternarysearch.java 

### PR DESCRIPTION
**Pull request by kknitk**
This Pull Request resolves Issue #8 

Description:
In this Pull Request we have done the following additions/edits for the code:

### CODE REVIEW

File: ternarysearch.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import java.util.Scanner;
```
Comment (MJ10) [inline ternarysearch.java]: Rename the file to your github username and move it to a directory named `Ternary_Search`

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by salman-bhai** [comment]
@MJ10 please review!

File: sieve_of_erastothenes/sieve_of_erastothenes.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#include <bits/stdc++.h>
```
Comment (MJ10) [inline sieve_of_erastothenes/sieve_of_erastothenes.cpp]: Rename the file to your github username

File: kknitk.java [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (MJ10) [inline kknitk.java]: Delete this file

File: ternary search/kknitk.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
+
```
Comment (MJ10) [inline ternary search/kknitk.java]: This is not a part of any existing issue. Please create a new issue on the repository and then reference that issue in the PR description

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR?

**Pull request closed without merging**